README.md

```diff
@@ -1,57 +1,53 @@
 # sigstore-java
 A sigstore java client for interacting with sigstore infrastructure
```

Comment:
Can we emphasize the alpha-ness and that things are subject to significant change and breakage?

Reply:
👍